lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -3192,7 +3210,7 @@ void HdVP2Material::CompiledNetwork::_UpdateShaderInstance(
 
     const bool matIsTransparent = _IsTransparent(mat);
     if (matIsTransparent != _surfaceShader->isTransparent()) {
-        _surfaceShader->setIsTransparent(matIsTransparent);
+        SetShaderIsTransparent(matIsTransparent);
```

Comment:
Transparency was previously only being set for the surface shader (whereas all other properties were set for all shader variants), which caused issues when using the front face shader.